examples/bidirectional/main.go

```diff
@@ -36,14 +36,14 @@ func main() {
 	rpcClient, err := client.Client()
 	if err != nil {
 		fmt.Println("Error:", err.Error())
-		os.Exit(1)
+		return
```

Comment:
What is the logic behind switching to `return`? We lose the error code when we don't use `os.Exit()`.

That said, this is only example code.